app/views/users/_form.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="page-body">
+  <div class="container-xl">
+    <div class="row row-deck row-cards">
+      <div class="col-12">
+        <div class="card">
+          <div class="card-body">
+            <br />
+            <%= form_with(model: User) do |f| %>
+              <div class="mb-3">
+                <%= f.label :first_name, class: 'form-label required' %>
+                <%= f.text_field :first_name, class: 'form-control' %>
+              </div>
+              <div class="mb-3">
+                <%= f.label :last_name, class: 'form-label required' %>
+                <%= f.text_field :last_name, class: 'form-control' %>
+              </div>
+              <div class="mb-3">
+                <%= f.label :email, class: 'form-label' %>
+                <%= f.text_field :email, class: 'form-control' %>
+              </div>
+              <div class="mb-3">
+                <%= f.label :admin, class: 'form-check-label' %>
+                <%= f.text_field :admin, class: 'form-check-input', type: 'checkbox' %>
```

Comment:
Szerintem működik, de légyszi ne így csináljuk. Létezik `checkbox` függvény a formon. Itt használjuk inkább azt a `text_field` + type specification helyett.
https://apidock.com/rails/ActionView/Helpers/FormHelper/check_box

Szerintem azért jelenhet meg neked helyesen, mert a `text_field` is egy `<input />` tag-et hoz létre, viszont te felülírod a `type` HTML attribútumát.